zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/DataVisualisationZip.java

```diff
@@ -180,16 +181,19 @@ private void unzipContent(ContentWriter contentWriter, Resource zipRes, String z
             ZipEntry zipEntry = zipInputStream.getNextEntry();
             Path filePath;
 
+            List<String> filesWritten = new ArrayList<>();
             while (zipEntry != null) {
 
                 if (isValidDataVisContentFile.test(zipEntry)) {
                     filePath = zipDir.resolve(zipEntry.getName());
                     contentWriter.write(zipInputStream, filePath.toString());
-                    info().data("zipPath", zipPath).log(DELETING_ZIP_DEBUG);
+                    filesWritten.add(filePath.toString());
```

Comment:
I see you've removed the misleading `DELETING_ZIP_DEBUG` log line, but there's another one at L153 that I think could be changed too.